src/QandaAtom.tsx

```diff
@@ -48,25 +48,25 @@ const bodyStyling = css`
     p {
```

Comment:
I suspect you want to refactor the Q&A atom and move the footer and summary components to be shared

Reply:
Yeah, I want to get the guide done first and merged then refactor Q and A

Reply:
OK! Yes, take my other comments with that in mind then 👍